docs/releases/5.0.txt

```diff
@@ -129,6 +129,11 @@ sets a database-computed default value. For example::
         created = models.DateTimeField(db_default=Now())
         circumference = models.FloatField(db_default=2 * Pi())
 
+Moving model between apps
+-------------------------
+
+Django now supports moving models between apps.
+
```

Comment:
Since no new operations are added maybe we could augment the description to mention that it's actually support for auto-detection of model move that was added?

```suggestion

The migration framework now auto-detect moving models between apps and supports
doing so without database operations if associated database table remains the same.
```